docs/config.rst

```diff
@@ -62,13 +111,76 @@ Java Kafka clients.
   * Default: ""
   * Importance: medium
 
+``simpleconsumer.cache.max.records``
+  Maximum number of records that can be stored for a specific topic-partition combination. Records with higher offsets replace records with lower ones Must be greater that 0.
+
+  * Type: int
+  * Default: 1000
+  * Importance: medium
+
+``simpleconsumer.max.caches.num``
```

Comment:
I agree that it is hard to calculate maximum size of RAM needed for rest proxy in the worst case. Even with number of caches property as we don't know size of each record. I will remove number of caches bound.